### PR TITLE
[QT-304] Ensure Chrome is only installed for vault-enterprise UI Test workflows

### PR DESCRIPTION
**Pull request by mikebaum**
After testing triggering the UI Tests GitHub Action manually, I noticed two problems:
* Chrome was being installed when the action was run in the `vault` repo, with the custom runner, which was not necessary
* The `ui_test_filter` was not properly set to `!enterprise` in the case when no filter was provided to the workflow.

This PR fixes these two issues, by:
* Changing the criteria for installing Chrome to rather check what repository the workflow is running in. Chrome should only be installed in the `vault-enterprise` repository.
* Updating the local used to set the ui test filter in the scenario to check if the filter is an empty string and if so adding the `!enterprise` filter.

### CODE REVIEW

File: .github/workflows/test-enos-scenario-ui.yml [hunk 1]
```diff
@@ -100,15 +99,16 @@ jobs:
         if: contains(${{ github.event.repository.name }}, 'ent')
         run: echo "${{ secrets.VAULT_LICENSE }}" > ./enos/support/vault.hclic || true
       - name: Install Chrome Dependencies
-        if: ${{ !needs.get-metadata.outputs.chrome-installed }}
+        if: ${{ needs.get-metadata.outputs.vault_edition == 'ent' }}
```
Comment (jaymalasinha) [inline .github/workflows/test-enos-scenario-ui.yml:102]: Do we want to also add the condition to check if chrome is already installed incase of re-run ?

File: .github/workflows/test-enos-scenario-ui.yml [hunk 2]
```diff
@@ -100,15 +99,16 @@ jobs:
...
         run: |
           sudo apt update
           sudo apt install -y libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev libasound2
       - name: Install Chrome
-        if: ${{ !needs.get-metadata.outputs.chrome-installed }}
+        if: ${{ needs.get-metadata.outputs.vault_edition == 'ent' }}
```
Comment (jaymalasinha) [inline .github/workflows/test-enos-scenario-ui.yml:107]: same suggestion as above to check for both conditions

File: .github/workflows/test-enos-scenario-ui.yml [hunk 3]
```diff
@@ -100,15 +99,16 @@ jobs:
...
         uses: browser-actions/setup-chrome@v1
-      - run: |
-         chrome --version
+      - name: Chrome Version
```
Comment (jaymalasinha) [inline .github/workflows/test-enos-scenario-ui.yml:109]: why did we need to separate this step ?

File: .github/workflows/test-enos-scenario-ui.yml [hunk 4]
```diff
@@ -135,7 +135,6 @@ jobs:
         id: run_ui_tests
```
Comment (jaymalasinha) [inline .github/workflows/test-enos-scenario-ui.yml:135]: This step doesn't run anything, so should be removed ?

**Review by jaymalasinha — CHANGES_REQUESTED** [review]

File: .github/workflows/test-enos-scenario-ui.yml (hunk 1 above)
Comment (mikebaum) [reply .github/workflows/test-enos-scenario-ui.yml:102]: yeah, good point... originally I had a check that checked if chrome was installed and it didn't seem to work. I can take a moment to revisit that approach and see if I can get it to work.

File: .github/workflows/test-enos-scenario-ui.yml (hunk 3 above)
Comment (mikebaum) [reply .github/workflows/test-enos-scenario-ui.yml:109]: I guess it's only useful to print the version of Chrome that is installed. Not sure that's logged anywhere else. Could be interesting to know that, if the UI tests fail for some weird reason, then you can test locally with that exact version of Chrome.

File: .github/workflows/test-enos-scenario-ui.yml (hunk 1 above)
Comment (mikebaum) [reply .github/workflows/test-enos-scenario-ui.yml]: updated to use a check that installs chrome if chrome is not installed.

File: .github/workflows/test-enos-scenario-ui.yml (hunk 2 above)
Comment (mikebaum) [reply .github/workflows/test-enos-scenario-ui.yml]: updated to use a check that installs chrome only if it is not already installed.

File: .github/workflows/test-enos-scenario-ui.yml (hunk 4 above)
Comment (mikebaum) [reply .github/workflows/test-enos-scenario-ui.yml]: Fixed.

File: .github/workflows/test-enos-scenario-ui.yml [hunk 5]
```diff
@@ -36,6 +36,7 @@ jobs:
       node-version: ${{ steps.get-metadata.outputs.node-version }}
       runs-on: ${{ steps.get-metadata.outputs.runs-on }}
       vault_edition: ${{ steps.get-metadata.outputs.vault_edition }}
+      chrome-version: ${{ steps.get-metadata.outputs.chrome-version }}
```
Comment (jaymalasinha) [inline .github/workflows/test-enos-scenario-ui.yml:39]: We don't need chrome version here anymore, correct ?

**Review by jaymalasinha — APPROVED** [review]
🏆

**Pull request merged**